Helm/opensearch-dashboards/values.yaml

```diff
@@ -59,23 +58,21 @@ securityContext:
   runAsNonRoot: true
   runAsUser: 1000
 
-
 config:
-  dashboards.yml:
+  opensearch_dashboards.yml:
```

Comment:
Ah a major miss. This [link](https://hub.docker.com/r/opensearchproject/opensearch-dashboards) was missed which is the reason it was not working. Never expected we will have `_` or custom yaml 🤕